server/src/main/resources/application-local.yml

```diff
@@ -38,3 +38,29 @@ datasources:
     password: 'postgres'
     schema-generate: CREATE_DROP
     dialect: POSTGRES
+
+micronaut:
+  security:
+    enabled: true
+    authentication: idtoken
+    oauth2:
+      clients:
+        google:
+          client-id: ${ OAUTH_CLIENT_ID }
```

Comment:
@mkimberlin how will we handle this locally ? as these are set in workflow and we don't run a workflow locally so how do we get the values ?